### ISSUE TITLE
font size

### ISSUE THREAD

**Issue by nsafa** [dunovank/jupyter-themes#271]
I wanna use cl options such as font and fontsize without changing default theme. Is it possible?
for instance I try $ jt -ofs 10, but I had no any changes!!

**Comment by boffi** [comment]
Just another user... if you specify a theme on the command line `jt` installs CSS, JS etc files in the notebook configuration folders, and those files are generated on the fly taking into account the font options that you have specified (e.g., `-ofs 10`).  

On the other hand, if you reset the theme (`jt -r`) or do not specify a theme `jt` doesnt install custom configuration, it just leaves the configuration folders in _status quo ante_.

TLDR no, afaict it's impossible
